benefits/static/css/styles.css

```diff
@@ -125,6 +132,11 @@ a[target="_blank"]::after {
   vertical-align: text-bottom;
 }
 
+.form-text a[target="_blank"]::after {
+  width: 18px;
+  height: 18px;
```

Comment:
What are these rules doing? Do we want to hardcode this pixel value?

Reply:
This is to make the external link icon smaller for the reCAPTCHA sentence. It's saying, any external link with the `form-text` class (which makes the font size 18px), the external link image should also be 18px. I can variable-ize it to make it easier to understand.

Reply:
Added variables and made a small CSS fix to get the icon to be the right size. https://github.com/cal-itp/benefits/pull/2586/commits/a4e9b6f15b0631970830be1c3766c9352f8909a6 to make the number meaning more explicit.

<img width="756" alt="image" src="https://github.com/user-attachments/assets/b654102a-e712-4ada-8af9-e2ca9f666625" />